cadquery/occ_impl/assembly.py

```diff
@@ -140,7 +140,7 @@ def toCAF(
     mesh: bool = False,
     tolerance: float = 1e-3,
     angularTolerance: float = 0.1,
-) -> Tuple[TDF_Label, TDocStd_Document]:
+) -> TDocStd_Document:
```

Comment:
Why change the signature, you can still return the top level label

Reply:
I see.  I may not have  considered potential external usage of the function.   Internal usage was ignoring the label.  Yes, this change could be reverted to restore the original signature.

Reply:
oops - not to mention breaking cq-editor.

Reply:
I'm getting the label from XCAFPrs_DocumentExplorer, however, it could instead by set inside _toCAF if preferred.

Reply:
I'd propose to return it from `_toCAF` and do not use any additional methods.